astro/src/content/docs/lifecycle/authenticate-users/logout-session-management.mdx

```diff
@@ -0,0 +1,516 @@
+| Works with non browser based applications, such as APIs | Yes | No |
+| Session revocation webhooks available | Yes | No |
+| Can be used without using the hosted login pages | Yes | No |
+
+Let's examine each of these approaches in more detail.
+
+## Centralized Sessions
+
+This section documents how to implement a centralized session store using FusionAuth. Let's use the example ChangeBank and ChangeBank Forum applications mentioned above.
+
+### Implementation Steps
+
+* Make sure to set up the ChangeBank and ChangeBank Forum Applications are correctly configured inside FusionAuth. The Application has the <InlineField>Refresh token</InlineField> checkbox toggled in the <InlineUIElement>Enabled Grants</InlineUIElement> section. The <InlineField>Generate refresh tokens</InlineField> setting must be enabled too.
+![Configuration of an application to allow the Refresh grant to be used.](/img/docs/lifecycle/authenticate-users/application-configuration-centralized-session-oauth.png)
+* Each time you build the authorization URL, your `scope` parameter must include the following scopes: `offline_access`, which creates the refresh token and `openid`, which asks for an Id token. The authorization URL will therefore include this string: `&scope=openid%20offline_access`, since scopes are separated with a URL encoded space.
+* In the ChangeBank and ChangeBank Forum applications, create an application session following your web or mobile app framework documentation.
+* After the user is logged in, store the refresh token Id in the application session. The Id is available in the `sid` parameter in the Id token, or in the access token. Prefer the refresh token Id instead of the refresh token value, since the Id will not change even if the value of the token itself does. This can happen if you are using one-time refresh tokens.
+  * You could also store the refresh token Id in a HttpOnly cookie instead of in a session.
+* Whenever the ChangeBank or ChangeBank Forum application receives a request, call the [Retrieve Refresh Tokens API](/docs/apis/jwt#retrieve-refresh-tokens), using the stored refresh token Id, to check the status of the refresh token. You can call the REST API directly or using one of [the FusionAuth client libraries](/docs/sdks). If the refresh token doesn't exist or is invalid, deny the user access and invalidate the application session.
+* When a user logs out from either the ChangeBank or ChangeBank Forum application, revoke the refresh token. Use the [Revoke Refresh Tokens API](/docs/apis/jwt#revoke-refresh-tokens). You can revoke all refresh tokens or just a few. The logic around revocation depends on your business needs. See [Flexible Revocation](#flexible-revocation) for more details.
+
+You are using the refresh token to tie the ChangeBank and ChangeBank Forum applications to FusionAuth. Your applications are then confirming session liveness with FusionAuth each time a request is received.
```

Comment:
```suggestion
You are using the refresh token to tie the ChangeBank and ChangeBank Forum applications to FusionAuth. Your applications are then confirming session validity with FusionAuth each time a request is received.
```

Reply:
Didn't accept your suggestion, but did tweak the language. Let me know if it works.